package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@adobe/aio-cli-plugin-api-mesh",
-	"version": "5.3.0-beta.2",
+	"version": "5.4.0",
```

Comment:
The [current stable](https://www.npmjs.com/package/@adobe/aio-cli-plugin-api-mesh?activeTab=versions) is 5.2.3 so this should be 5.3.0 instead